doctr/models/detection/differentiable_binarization/pytorch.py

```diff
@@ -231,8 +239,8 @@ def compute_loss(self, out_map: torch.Tensor, thresh_map: torch.Tensor, target:
             A loss tensor
         """
 
-        prob_map = torch.sigmoid(out_map.squeeze(1))
-        thresh_map = torch.sigmoid(thresh_map.squeeze(1))
+        prob_map = torch.sigmoid(out_map)
+        thresh_map = torch.sigmoid(thresh_map)
```

Comment:
Why is `squeeze` not needed anymore ?

Reply:
they were squeezing the classes shape as it was always 1, we need it now